lib/core/app/app_router.dart

```diff
@@ -25,7 +25,7 @@ import 'package:go_router/go_router.dart';
 
 class AppRouter {
   static final _rootNavigatorKey = GlobalKey<NavigatorState>();
-  static bool ignoreCoinFlow = false;
+  static bool inAppCoinFlow = false;
```

Comment:
can we remove this flag and use FlowsCubit state instead? 
we can add getter to the FlowsState like: 
`bool get isInAppCoin {
    return flowType == FlowType.inAppCoin;
  }
`

Reply:
Yes fair, done